pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix

```diff
@@ -95,6 +95,7 @@ let
     x86_64-linux = fetchurl {
       urls = [
         "https://dist.torproject.org/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
+        "https://web.archive.org/web/20220916130350/https://tor.eff.org/dist/torbreawser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
```

Comment:
Not sure if that's a good idea: when updating the version now, the `web.archive.org` link will be brokne because of the "timestamp", correct?

Not sure what would be a better solution here, though.

Reply:
We can request the date ``*`` but then it still needs to be uploaded to web.archive.org before deletion.

Reply:
One could write a checklist on what to do when updating this package which includes that.

Reply:
Would a comment on the timestamp be good enough?